internal/locate/region_cache.go

```diff
@@ -1027,6 +1084,8 @@ func (c *RegionCache) BatchLoadRegionsWithKeyRange(bo *retry.Backoffer, startKey
 	c.mu.Lock()
 	defer c.mu.Unlock()
 
+	// TODO(youjiali1995): scanRegions always fetch regions from PD and these regions don't contain buckets information
```

Comment:
Does it mean that we are going to add option to the `ScanRegions` interface, so it can return buckets in the same response?

Reply:
No, too much network traffic. A 10GB region can contains 100 buckets, so the payload of a single region can be several KBs. I will make `BatchLoadRegionsWithKeyRange()` use `scanRegionsFromCache()` or not insert regions to cache.